h3i/examples/content_length_mismatch.rs

```diff
@@ -0,0 +1,74 @@
+use h3i::actions::h3::Action;
+use h3i::client::sync_client;
+use h3i::config::Config;
+use quiche::h3::frame::Frame;
+use quiche::h3::Header;
+use quiche::h3::NameValue;
+
+fn main() {
+    let config = Config::new()
+        .with_host_port("blog.cloudflare.com".to_string())
+        .with_idle_timeout(2000)
+        .build()
+        .unwrap();
+
+    let headers = vec![
+        Header::new(b":method", b"POST"),
+        Header::new(b":scheme", b"https"),
+        Header::new(b":authority", "blog.cloudflare.com".as_bytes()),
+        Header::new(b":path", b"/"),
+        // We say that we're going to send a body with 5 bytes...
+        Header::new(b"content-length", b"5"),
+    ];
+
+    let header_block = encode_header_block(&headers).unwrap();
```

Comment:
Let's explain what this does
```suggestion
    // HTTP/3 uses QPACK header compression. Our HEADERS frame carries the encoded
    // QPACK bytes.
    let header_block = encode_header_block(&headers).unwrap();
```